BraveRewardsUI/Settings/Grants/SettingsGrantSectionView.swift

```diff
@@ -50,10 +50,11 @@ class SettingsGrantSectionView: SettingsSectionView {
     switch type {
     case .ads(let amount):
       iconImageView.image = UIImage(frameworkResourceNamed: "icn-ads")
+      
       if let amount = amount {
-        textLabel.text = String(format: Strings.SettingsAdsGrantText, "\(amount) \(Strings.BAT) ")
+        textLabel.text = "\(amount) \(Strings.BAT) " + Strings.SettingsAdsGrantText
```

Comment:
This string is not formatted, there is a chance that it may not  sound correctly for all languages, we could always fall back to using the `NotificationEarningsClaimDefault` for all cases